wait_for_db.sh

```
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
```

Consider adding `set -u` (treat unset variables as an error) and/or validating inputs so the script fails early on missing/invalid arguments rather than continuing with empty variables. Right now `host` can be empty if no argument is passed.